Minor fixes to Chapter 15 (#1008)
browserengineering/book: book/embeds.md

*Canvas element*: Implement the [`<canvas>`][canvas-elt] element, the 2D aspect
of the [`getContext`][getcontext] API, and some of the drawing commands on
[`CanvasRenderingContext2D`][crc2d]. Canvas layout is just like an iframe,

> including its default width and height. You should allocate a Skia surfae of

Reviewer: ```suggestion
including its default width and height. You should allocate a Skia surface of
```

*Object-fit*: implement the [`object-fit`][obj-fit] CSS property. It determines
how the image within an `<img>` element is sized relative to its container

> element. This will require clipping images with the wrong aspect ratio.

Reviewer: ```suggestion
element. This will require clipping images with a different aspect ratio.
```
